app/controllers/bankdetails/BankAccountIsUKController.scala

```diff
@@ -81,7 +81,7 @@ class BankAccountIsUKController @Inject()(
               if (edit) {
                 Future.successful(Redirect(routes.SummaryController.get(false)))
               } else {
-                lazy val redirect = Redirect(routes.BankAccountRegisteredController.get(index))
+                lazy val redirect = Redirect(routes.YourBankAccountsController.get())
```

Comment:
Can you have a chat to Fi about this - it should be going to check your answers and have an index associated with it - but she may already have changed it.

Reply:
@nonimay ^^